packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -239,6 +250,26 @@ export class VaBreadcrumbs {
     });
   }
 
+   /**
+   * Emit event with payload equal to href of the link clinked
+   * Only fires if link is part of breadcrumb object that also has isLinkRouter = true property
+   */
+   handleRouteChange(e: MouseEvent, href: string): void {
+    e.preventDefault();
+    this.routeChange.emit({href});
+  }
+
+  /**
+   * Handle click event on breadcrumb when uswds is true
+   */
+  uswdsHandler(e: MouseEvent, breadcrumb: { href: string, isRouterLink?: boolean}): void {
```

Comment:
I think naming this `uswdsHandler` doesn't really explain what the method does in a future proof way. Like at some point there will be no "uswds" and "non-uswds" variations. Maybe just `handleClick`?